### PR TITLE
converted changelist.txt to markdown format

### PR DESCRIPTION
**Pull request by enricosada**
renamed changelog file name to `CHANGELOG.md`, see demo [on my branch](https://github.com/enricosada/visualfsharp/blob/changelog_md/CHANGELOG.md)

converted codeplex issue id to links

`FEATURE` for new features.
`CHANGED` for changes of behaviour in existing functionality.
`DEPRECATED` for once-stable features removed in upcoming releases.
`REMOVED` for deprecated features removed in this release.
`BUGFIX` for any bug fixes.
`ENHANCEMENT` for improvement to existing functionality 

rif #178

### CODE REVIEW

**Comment by forki** [comment]
Bonus points for making this compatible with projectscaffold format (or fixing the parser in FAKE)

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
+# Visual F<span>#</span>
+All notable changes to this project will be documented in this file.
+
+This project **don't** uses [Semantic Versioning](http://semver.org/)
```
Comment (mexx) [inline CHANGELOG.md]: Please update it to:
This project **doesn't** use [Semantic Versioning](http://semver.org/)

Comment (enricosada) [inline CHANGELOG.md]: done, thx

**Comment by enricosada** [comment]
updated! flattened structure more like projectscaffold.
@forki I'll check for compatibility of fake parser :D

`October 29 2013` => `2013-10-29` because not everyone use MDY date format (magenta)


source: [wikipedia](https://en.wikipedia.org/wiki/Date_format_by_country)

**Comment by enricosada** [comment]
or we can use :horse_racing: (perf) :bomb: (deprecated) :fire: (removed) :bug: (fix) 

just kidding

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+- `PERFORMANCE` Modified GC settings on the compiler for better performance
+- `FEATURE` Normalized collections modules
+
+### Visual Studio
+
+- `FEATURE` Assembly metadata in project templates
+- `PERFORMANCE` Improved startup time for FSI
+- `FEATURE` New hotkeys for FSI
+
+## [4.0.0] / 2014-08-20
```
Comment (latkin) [inline CHANGELOG.md]: This should be 3.1.2

Comment (enricosada) [inline CHANGELOG.md]: fixed :memo:

**Comment by forki** [comment]
how about adding links to the corresponding issues?

**Comment by enricosada** [comment]
i added some links, where the codeplex ticket was present. I tried to search some on codeplex but no luck, and boring (more than convert to markdown :smile:  )

File: CHANGELOG.md [hunk 3]
```diff
@@ -0,0 +1,95 @@
+# Visual F<span>#</span>
```
Comment (latkin) [inline CHANGELOG.md]: Can you maintain the copyright notice?

Comment (enricosada) [inline CHANGELOG.md]: sure, added back copyright notice

**Comment by latkin** [comment]
We don't want to file to be named differently in master and fsharp4, can you please rebase this on master so we can then merge it into fsharp4 after?

**Comment by enricosada** [comment]
i rebased to master, but the pr target branch cannot be modified

added #202

**Comment by forki** [comment]
> i rebased to master, but the pr target branch cannot be modified

why not?
usually we just `push -f` on the pull requests!?

**Comment by enricosada** [comment]
i have done that (was already rebased on master), but pr target (see on top) is not changed and cannot on github.

**Comment by forki** [comment]
Ah. I misread that. You changed the target branch.  Sorry

**Comment by enricosada** [comment]
np, and thx anyway, push to pr is the best trick i learned about github

**Comment by latkin** [comment]
Which PR are we using for this? Please close whichever one you don't want to maintain.

**Comment by latkin** [comment]
Closing this one out in favor of #202

**Pull request closed without merging**